CHANGELOG.md

```diff
@@ -4,13 +4,13 @@
 * None
 
 ### Fixed
-* None
+* Fixed corruption bugs when encryption is used. (Core Issue [#5360](https://github.com/realm/realm-core/issues/5360))
```

Comment:
The linked Core issue is actually still open. Was it just not closed or should we link a different issue?

Reply:
Down in the conversation it says it was fixed by https://github.com/realm/realm-core/pull/5383. At some point they'll need to close this ticket 😁